src/seedu/addressbook/commands/EditCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Edits the particulars of a person whose name matches exactly as typed in the command.
```

Comment:
Good documentation before the class implementation to have a overview of what the class will be doing.